exchangesync/node_policy.go

```diff
@@ -92,12 +92,19 @@ func GetProcessedExchangeNodePolicy(pDevice *persistence.ExchangeDevice, getExch
 		glog.V(2).Infof("Failed to retrieve node policy from local db: %v", err)
 	}
 
+	isCluster := false
+	exchDevice, err := persistence.FindExchangeDevice(db)
```

Comment:
Same comment, pDevice is already passed in. I dont think you need to read it again from the local DB.

Reply:
fixed